airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/testutils/PostgresTestDatabase.java

```diff
@@ -40,18 +40,51 @@ public class PostgresTestDatabase implements AutoCloseable {
 
   static private final Logger LOGGER = LoggerFactory.getLogger(PostgresTestDatabase.class);
 
+  public enum PostgresImage {
+
+    POSTGRES_12_BULLSEYE("postgres:12-bullseye"),
+    POSTGRES_16_BULLSEYE("postgres:16-bullseye"),
+    POSTGRES_16_ALPINE("postgres:16-alpine"),
+    POSTGRES_9_ALPINE("postgres:9-alpine"),
+    ;
+
+    private final String imageName;
```

Comment:
Suggestion: have the value be a DockerImageName instead of a String.

Reply:
I'm going to keep this one like this because we use the imageName as a string as a part of the LAZY key.